website/pages/docs/[[...slug]].tsx

```diff
@@ -128,7 +128,10 @@ export const getStaticProps: GetStaticProps = async ({ params }) => {
   const headings = getHeadings(source);
 
   const root = path.join(process.cwd(), '..', 'packages');
-  const packages = fs.readdirSync(root).filter(junk.not);
+  const packages = fs
+    .readdirSync(root)
+    .filter(junk.not)
+    .filter((file) => !/tsconfig\.json/.test(file));
```

Comment:
@sarahdayan this was a pretty odd error I ran into by adding a `tsconfig.json` to the packages directory. It was being loaded by the `BundleSize.tsx` file by mistake. This regex test isn't very elegant but works. Another solution would be to check where or not its a directory.

Reply:
That'll do for now 👍